ext/IntervalArithmeticsIntervalSetsExt.jl

```diff
@@ -0,0 +1,9 @@
+module IntervalArithmeticsIntervalSetsExt
+
+import IntervalSets as IS
+import IntervalArithmetic as IA
+
+Base.convert(T::Type{<:IS.Interval}, i::IA.Interval) = T(IA.inf(i), IA.sup(i))
+Base.convert(T::Type{<:IA.Interval}, i::IS.Interval) = T(IS.infinum(i), IS.supremum(i))
```

Comment:
This won't work.
It should probably be in the same vein as how we deal with `Real` number conversion, i.e.

```Julia
function Base.convert(::Type{Interval{T}}, i::IS.Interval) where {T<:NumTypes}
    x = IA.interval(T, IS.infinum(i), IS.supremum(i))
    return IA._unsafe_interval(IA.bareinterval(x), IA.decoration(x), false)
end
```